jest/jest.d.ts

```diff
@@ -1,126 +1,88 @@
-// Type definitions for Jest 0.9.0
+// Type definitions for Jest 15.1.1
 // Project: http://facebook.github.io/jest/
-// Definitions by: Asana <https://asana.com>
+// Definitions by: Asana <https://asana.com>, Ivo Stratev <https://github.com/NoHomey>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /// <reference path="../node/node.d.ts" />
 
-declare function afterEach(fn: jest.EmptyFunction): void;
-declare function beforeEach(fn: jest.EmptyFunction): void;
-declare function describe(name: string, fn: jest.EmptyFunction): void;
-declare var it: jest.It;
-declare function pit(name: string, fn: jest.EmptyFunction): void;
-
-declare function xdescribe(name: string, fn: jest.EmptyFunction): void;
-declare function xit(name: string, fn: jest.EmptyFunction): void;
-
+declare function afterEach(fn: () => any): void;
+declare function beforeEach(fn: () => any): void;
+declare function describe(name: string, fn: () => any): void;
 declare function expect(actual: any): jest.Matchers;
+declare function it(name: string, fn: () => any): void;
+declare function fit(name: string, fn: () => any): void;
 
-interface NodeRequire {
-    requireActual(moduleName: string): any;
-}
+declare function test(name: string, fn: () => any): void;
+declare function xdescribe(name: string, fn: () => any): void;
+declare function xit(name: string, fn: () => any): void;
 
 declare namespace jest {
-    function addMatchers(matchers: CustomMatcherFactories): void;
```

Comment:
This function does exist on the jest object. If we end up not bringing in jasmine types, it could just be typed as any.